OrbitGl/DataManager.h

```diff
@@ -13,11 +13,29 @@
 #include "TextBox.h"
 #include "absl/container/flat_hash_map.h"
 #include "absl/container/flat_hash_set.h"
+#include "tracepoint.pb.h"
 
 // This class is responsible for storing and
 // navigating data on the client side. Note that
 // every method of this class should be called
 // on the main thread.
+
+using orbit_grpc_protos::TracepointInfo;
+
+namespace internal {
+struct HashTracepointInfo {
+  size_t operator()(const TracepointInfo& info) const {
+    return std::hash<std::string>{}(info.category()) * 37 + std::hash<std::string>{}(info.name());
+  }
+};
+
+struct EqualTracepointInfo {
+  size_t operator()(const TracepointInfo& left, const TracepointInfo& right) const {
+    return left.category().compare(right.category()) == 0;
```

Comment:
I think you forgot to compare the name?

Reply:
and equal returns bool not size_t.

Reply:
+1 , should be something like
`left.category() == right.category() && left.name() == right.name()`